internal/slicer/slicer.go

```diff
@@ -397,3 +434,160 @@ func createFile(targetPath string, pathInfo setup.PathInfo) (*fsutil.Entry, erro
 		MakeParents: true,
 	})
 }
+
+type generateManifestsOptions struct {
+	packageInfo []*archive.PackageInfo
+	selection   []*setup.Slice
+	report      *Report
+	targetDir   string
+}
+
+func generateManifests(options *generateManifestsOptions) error {
```

Comment:
The earlier comment was about generateManifest itself, not just LocateManifestSlices. Per earlier thread, that's also why I was mentioning that the testing of what is in here can be done in the manifest, so we don't need to involve the slicer on everything.